test/uk/gov/hmrc/economiccrimelevyregistration/controllers/contacts/FirstContactEmailControllerSpec.scala

```diff
@@ -45,7 +45,7 @@ class FirstContactEmailControllerSpec extends SpecBase {
 
   val mockEclRegistrationConnector: EclRegistrationConnector = mock[EclRegistrationConnector]
 
-  val emailMaxLength: Int = 160
+  val emailMaxLength: Int = 132
```

Comment:
Should this be using uk.gov.hmrc.economiccrimelevyregistration.forms.mappings.MaxLengths.EmailMaxLength instead?